PSI-MOD.obo

```diff
@@ -12679,9 +12584,10 @@ xref: TermSpec: "none"
 xref: Unimod: "Unimod:159"
 xref: GNOme: "GNO:G28681TP"
 is_a: MOD:00725 ! complex glycosylation
+is_a: MOD:00160 ! N4-glycosyl-L-asparagine
 
 [Term]
-id: MOD:00528

```

Comment:
very small point, but the added extra space might throw off some parsers that aren't being careful